script/openqa-bootstrap

```diff
@@ -46,7 +46,12 @@ if ping -c1 download.suse.de. && (! rpm -q ca-certificates-suse) ; then
     if ! zypper info ca-certificates-suse | grep -q ':' ; then
         # add suse ca repo if needed
         # use this way of adding the repo to be distro agnostic
-        zypper -n addrepo obs://SUSE:CA SUSE:CA
+        if [ "$NAME" = "openSUSE Leap" ]; then
+            # avoid using `obs://…` URL to workaround https://bugzilla.opensuse.org/show_bug.cgi?id=1187425
+            zypper -n addrepo "https://download.suse.de/ibs/SUSE:/CA/openSUSE_Leap_${VERSION}" 'SUSE:CA'
```

Comment:
We can't use `https` before the ca is added.

```suggestion
            zypper -n addrepo "http://download.suse.de/ibs/SUSE:/CA/openSUSE_Leap_${VERSION}" 'SUSE:CA'
```

Reply:
In my test it worked: https://paste.opensuse.org/36389794

However, I have no idea how. Maybe it is better to use just http instead of relying on some magic I don't understand.